src/_includes/common/errors-context.md

```diff
@@ -0,0 +1,186 @@
+    
+    {{ include.errors_configure_tags }}
+
+    [{% asset additional-data/tags.png alt="All tags and their related values." %}]({% asset additional-data/tags.png @path %})
+
+- *Context* includes additional diagnostic information attached to an event. By default, contexts are not searchable, but for convenience Sentry turns information in some predefined contexts into tags, making them searchable.
+
+   [{% asset additional-data/predefined_contexts.png alt="Predefined data such as the name and version." %}]({% asset additional-data/predefined_contexts.png @path %})
+
+## What Sentry Sends Automatically
+
+Certain data is sent to Sentry automatically. This section explains the predefined data, some of which is simply the type of device or browser being used at the time of the event. Other predefined data you can modify, such as the `level` of the event. The final type of predefined data, `environment` and `release`, affect the UI experience or enable features within the product to help better identify issues in your application.
+
+### Predefined Data
+
+Sentry turns additional, predefined data or specific attributes on the data into _tags_, which you can use for a variety of purposes, such as searching in the web UI for the event or delving deeper into your application. For example, you can use tags such as `level` or `user.email` to surface particular errors. You can also enable Sentry to track [releases](/workflow/releases/), which unlocks features that help you delve more deeply into the differences between deployed releases.
+
+If Sentry captures some predefined data but doesn’t expose it as a tag, you can set a custom tag for it.
+
+Request, device, OS, runtime, app, browser, GPU, logger, and monitor are the most typical predefined data sent with an event. In addition, the following are sent, and can be modified for your team's use:
+
+Level
+
+: Defines the severity of an event. The level can be set to one of five values, which are, in order of severity: `fatal`, `error`, `warning`, `info`, and `debug.error`. Learn how to set the level in Set the Level
+
+User
+
+: Providing user information to Sentry helps you evaluate the number of users affecting an issue and evaluate the quality of the application. Learn how to capture user information in Capture the User
+
+Fingerprint
+
+: Sentry uses one or more fingerprints to determine how to group errors into issues. Learn more about Sentry's approach to grouping algorithms and fingerprinting in Grouping & Fingerprints. Learn how to override the the default group in very advanced use cases in Modifying the Default Fingerprint
+
+Environment
+
+: Environments help you better filter issues, releases, and user feedback in the Issue Details page of the web UI. Learn how to set and manage environments in Manage Environments
```

Comment:
Is "Manage Environments" supposed to be linked to a page or section?

Reply:
yes. fixed. Thanks.